### ISSUE TITLE
Тригер второй волны на Forgotten Core Information Pen

### ISSUE THREAD

**Issue by Gamer-Grad** [minlexx/whdbx_web#21]
Добавил немного информации по Forgotten Core Information Pen

https://github.com/Gamer-Grad/whdbx_web/commit/581c80d85585d8d642280a52074a26e0dcd1fd34#diff-81fa1ed7c4d10c51485bc79e0aed5d7b

Строки с 44 по 52, html комментарий русский нужно убрать

Строка 130 и 135, небольшая css обертка чтоб выделить важную информацию

https://github.com/Gamer-Grad/whdbx_web/commit/581c80d85585d8d642280a52074a26e0dcd1fd34#diff-c66924bbd834df4846001110523c1276

Строки 927 и 937, локализация под Forgotten Core Information Pen

С системами контроля версий особо не работал, комит кривоватый вышел...

**Comment by Gamer-Grad** [comment]
Выглядеть будет так https://wh.sib-eve.ru/signatures/38/

**Comment by minlexx** [comment]
Так, подожди, идея здравая, но тот факт что для триггера волны на слиперских дата-реликах надо ломать конты - это давно известный факт и не только для `Forgotten Core Information Pen` это верно, а также и для многих других, чуть ли не для каждого дата/релик сайта начиная с с4 класса и выше (с4, с5, с6)

Рациональнее было бы добавить такой комментарий под все дата/релик сайты начиная с 4го класса, не думаешь? 

Вообще там ситуация довольно очевидная, если волна следующая не спавнится - значит надо ломать конты.

**Comment by Gamer-Grad** [comment]
Хм... Есть аналогичный релик в С4 [Forgotten Frontier Conversion Module](https://wh.sib-eve.ru/signatures/29/), а вот в  С6, везде есть тригеры.

Еще в С2 датка [Unsecured Perimeter Transponder Farm ](https://wh.sib-eve.ru/signatures/16/)

Все больше нету.

У меня решение больше такое, костыльное, было бы проще написать функцию проверки наличия тригера, которой в параметры втыкать ид аномали, порядковый номер волны и наличие тригера.

Но реально таких аномали всего 3.

**Comment by minlexx** [comment]
Можно добавить комментарий под все дата/релики что если следующей волны нет, надо ломать конты, и все, эта логика никогда подводила меня до сих пор))
